trunner/tools/phoenix.py

```diff
@@ -22,6 +22,23 @@ def wait_for_dev(port, timeout=0):
             raise TimeoutError
 
 
+def wait_for_vid_pid(port_hint: str, timeout=0):
+    asleep = 0
+    port = None
+
+    while port is None:
+        time.sleep(0.01)
+        asleep += 0.01
+
+        for p in list_ports.grep(port_hint):
```

Comment:
grep uses regexp as a param, either make a regexp or search for vid/pid by looping (they are properties of [ListPortInfo](https://pyserial.readthedocs.io/en/latest/tools.html#serial.tools.list_ports.ListPortInfo)).

Also - don't iterate over results (if there is more than one it should be considered an error?)

Reply:
I'm not 100% sure about the solution I used, so I'll wait for your feedback